portal/templates/profile_macros.html

```diff
@@ -2072,7 +2082,28 @@ <h4 class="modal-title">{{ _("Enter questionnaire manually on patient's behalf")
                             };
 
                             if (!hasValue(errorMsg)) {
-                                continueToAssessment(method, completionDate);
+                                var linkUrl = "";
+                                /*
+                                 * retrieve instrument ids from present questionnaire bank
+                                 */
+                                if (data.questionnaire_bank.questionnaires) {
```

Comment:
Is this still going to work without the `resume_instrument_id` QS params?

Reply:
@ivan-c Ivan, the resume_instrument_id info is contained in the assessment link.  I am merely appending the instrument id list to id.